ast-visual-studio-extension/CxExtension/Panels/ResultVulnerabilitiesPanel.cs

```diff
@@ -143,6 +162,121 @@ private void BuildVulnerabilityLocation()
             vulnerabilitiesList.Items.Add(item);
         }
 
+
+        /// <summary>
+        /// Fill Learn More and Remediation Examples tab
+        /// </summary>
+        /// <param name="cxToolbar"></param>
+        public async Task LearnMoreAndRemediationAsync(CxToolbar cxToolbar)
+        {
+            CxCLI.CxWrapper cxWrapper = CxUtils.GetCxWrapper(cxToolbar.Package, cxToolbar.ResultsTree, GetType());
+
+            if (learnMore != null) return;
+
+
+            cxWindowUI.LearnMorePanelTitle.Children.Clear();
+            cxWindowUI.RemediationPanelTitle.Children.Clear();
+
+            AddTextWithTitle(cxWindowUI.LearnMorePanelTitle, Utils.CxConstants.LOADING_INFORMATION);
+            AddTextWithTitle(cxWindowUI.RemediationPanelTitle, Utils.CxConstants.LOADING_INFORMATION);
+
+            if (cxWrapper == null) return;
+                await Task.Run(() =>
+                {
+                    try
+                    {
+                        learnMore = cxWrapper.LearnMoreAndRemediation(result.Data.QueryId);
+
+                    }
+                    catch (Exception ex)
+                    {
+                        CxUtils.DisplayMessageInInfoBar(cxToolbar.Package, string.Format(Utils.CxConstants.ERROR_GETTING_LEARNMORE, ex.Message), KnownMonikers.StatusError);
+                    }
+                });
+            cxWindowUI.LearnMorePanelTitle.Children.Clear();
+            cxWindowUI.RemediationPanelTitle.Children.Clear();
```

Comment:
Once we're cleaning these panels after checking if the learnMore != null, we can remove it from here.

Reply:
Actually, we need to Clear the panels because we are printing "Loading information..." before.